.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -10,23 +10,62 @@ on:
         description: "The git ref to check out from the specified repository."
         required: false
         default: master
+      part-to-bump:
```

Comment:
nit: i'd rename to `release type` or something since the release itself is either major/minor/bump